### PR TITLE
Azure Core October Release

### PR DESCRIPTION
**Pull request by antkmsft**
Live tests for Core, Identity, Storage, KeyVault, Attestation - all pass.
Vcpkg also does pass: https://dev.azure.com/vcpkg/public/_build/results?buildId=78964&view=results

### CODE REVIEW

File: sdk/core/azure-core/CHANGELOG.md [hunk 1]
```diff
@@ -19,10 +19,6 @@
 
   If the original behavior is desired, customers can adjust these timeouts by changing the `RetryDelay` and `MaxRetryDelay` fields in in the `TransportOptions` structure.
```
Comment (ahsonkhan) [inline sdk/core/azure-core/CHANGELOG.md:20]: Please fix the two typos:
```suggestion
  If the original behavior is desired, customers can adjust these timeouts by changing the `RetryDelay` and `MaxRetryDelay` fields in the `RetryOptions` structure.
```

FYI @LarryOsterman, those fields are in the `RetryOptions, and not the `TransportOptions`:
https://github.com/Azure/azure-sdk-for-cpp/blob/53c9da155244b72ac90ad27f515d4ea1e7505a01/sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp#L88-L100

**Review by ahsonkhan — APPROVED** [review]
Otherwise, looks good.

File: sdk/core/azure-core/CHANGELOG.md (hunk 1 above)
Comment (antkmsft) [reply sdk/core/azure-core/CHANGELOG.md]: Thank you Ahson! Very good catch!

**Review by LarryOsterman — APPROVED** [review]

**Pull request merged**